.github/workflows/klocwork-pull-request.yml

```diff
@@ -62,12 +65,16 @@ jobs:
         run: kwinject --output kwinject.out ninja -C build -v -k0
 
       - name: fully analyze build of base ref
-        run: kwcheck run --build-spec kwinject.out -F scriptable --report issues.txt --project-dir /home/klocwork/.kwlp --settings-dir /home/klocwork/.kwps
+        run: kwcheck run --build-spec kwinject.out -F scriptable --report /home/klocwork/parent_issues.txt --project-dir /home/klocwork/parent/.kwlp --settings-dir /home/klocwork/parent/.kwps
 
-      - name: ignore issues of base ref
+      # Use non numeric sort because comm expect default sorting order
+      - name: get parent issue ids
+        run: awk -F ';' '{print $1}' /home/klocwork/parent_issues.txt | sort > /home/klocwork/parent_issues_ids.txt
+      
+      - name: copy parent klocwork project
         run: |
-          count=$(wc -l < issues.txt)
-          test "$count" = "0" || kwcheck set-status 1-"$count" --status ignore --project-dir /home/klocwork/.kwlp
+          cp -r /home/klocwork/parent/.kwlp /home/klocwork/child/.kwlp 
+          cp -r /home/klocwork/parent/.kwps /home/klocwork/child/.kwps
```

Comment:
This may be combined:
```
cp -r /home/klocwork/parent/.kwlp /home/klocwork/parent/.kwps /home/klocwork/child
```
(Bash would also allow for
```
cp -r /home/klocwork/parent/.kw{lp,ps} /home/klocwork/child
```
which is nice for interactive use, but in scripts I would stick to the first, portable form.)